src/ContractInteractor.ts

```diff
@@ -93,6 +97,7 @@ export default class ContractInteractor {
     private readonly IRelayHubContract: Contract<IRelayHubInstance>;
     private readonly IForwarderContract: Contract<IForwarderInstance>;
     private readonly IWalletFactoryContract: Contract<IWalletFactoryInstance>;
+    private readonly ERC20Contract: Contract<ERC20Instance>;
```

Comment:
Probably this should be an interface as the others, so I'd rename it to `IERC20Contract`

Reply:
it has the name of the Artifact that is used, Im using the following Artifact:

[ERC20](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.2.0/contracts/token/ERC20/ERC20.sol)

The other has an I since are Interfaces but this is not an interface.